src/UKHO.ERPFacade.Common/Models/PriceChange.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace UKHO.ERPFacade.Common.Models
+{
```

Comment:
Remove unused namespaces and kindly exclude this code from code coverage

Add [ExcludeFromCodeCoverage]

Reply:
Added [ExcludeFromCodeCoverage]